deployment-guide.md

```diff
@@ -1,11 +1,21 @@
 # Bonsai Deployment Guide
+
 > **Note: This software is not production ready. Do not use in production.**
 
-Welcome to the [Bonsai] Deployment guide! 
+Welcome to the [RISC Zero] Ethereum Deployment guide!
+
+Once you've written your [contracts] and your [methods], and [tested] your program, you're ready to deploy your contract.
+
+You can either:
+
+- [Deploy your project to a local network]
+- [Deploy to a testnet]
 
-Once you've written your [contracts] and your [methods], and [tested] your program, you're ready to deploy your contract. You can either:
-- Deploy your project to a local network
-- Deploy to a testnet
+[methods]: ./methods/
+[contracts]: ./contracts/
+[tested]: ./README.md#run-the-tests
+[Deploy your project to a local network]: #deploy-your-project-on-a-local-network
+[Deploy to a testnet]: #deploy-your-project-on-a-testnet
```

Comment:
All links go to the bottom of a file, this makes it much easier to adjust links in the future

Reply:
Interesting, I've always thought the opposite. I'll make sure to put them at the bottom of the page for now on 👍

Reply:
When I go to update links, if I can just scan all the links at the bottom it makes it a lot easier to fix and update if I don't have to go hunting for all the links in the content.